docs/deployment.md

```diff
@@ -43,7 +43,8 @@ machine where all Mesh-related data will be stored.
 -   Ports 60557, 60558, and 60559 are the default ports used for the JSON RPC endpoint, communicating with peers over TCP, and communicating with peers over WebSockets, respectively.
 -   In order to disable P2P order discovery and sharing, set `USE_BOOTSTRAP_LIST` to `false`.
 -   Running a VPN may interfere with Mesh. If you are having difficulty connecting to peers, disable your VPN.
--   If you are running against a POA testnet (e.g., Kovan), you might want to shorten the `BLOCK_POLLING_INTERVAL` since blocks are mined more frequently then on mainnet.
+-   If you are running against a POA testnet (e.g., Kovan), you might want to shorten the `BLOCK_POLLING_INTERVAL` since blocks are mined more frequently then on mainnet. If you do this, your node will use more Ethereum RPC calls, so you will also need to adjust the `ETHEREUM_RPC_MAX_REQUESTS_PER_24_HR_UTC` to around `380000` (*warning:* this will not fit within Infura's
```

Comment:
Since you didn't specify how much to reduce `BLOCK_POLLING_INTERVAL` by, including a specific number that you would need for `ETHEREUM_RPC_MAX_REQUESTS_PER_24_HR_UTC` doesn't make a lot of sense. Can we just warn about setting `ETHEREUM_RPC_MAX_REQUESTS_PER_24_HR_UTC` higher than 100k?